Tools/CISupport/ews-build/steps.py

```diff
@@ -213,17 +215,35 @@ def get_reviewers(self, pr_number, repository_url=None):
         if not api_url:
             return []
 
-        reviews_url = f'{api_url}/pulls/{pr_number}/reviews'
-        content = self.fetch_data_from_url_with_authentication_github(reviews_url)
-        if not content:
-            return []
+        reviews = []
+        reviews_url = f'{api_url}/pulls/{pr_number}/reviews?per_page={self.PER_PAGE_LIMIT}'
+        page = 1
+        while page < self.NUM_PAGE_LIMIT + 1:
+            url = f'{reviews_url}&page={page}'
+            content = self.fetch_data_from_url_with_authentication_github(url)
+            if not content:
+                break
+            response_content = content.json() or []
+            if not isinstance(response_content, list):
+                self._addToLog('stdio', f"Malformed response when listing reviews with '{url}'\n")
+                break
+            reviews += response_content
+            if self.PER_PAGE_LIMIT < len(response_content):
```

Comment:
This still seems inverse. Maybe I am missing something.

Reply:
Nope, my brain just wasn't working...updating